Remove GNotification support added in https://bugzilla.gnome.org/show_bug.cgi?id=747146
Reverts:
https://git.gnome.org/browse/glib/commit/?id=89058e8a9b769ab223bc75739f5455dab18f7a3d
And most of:
https://git.gnome.org/browse/glib/commit/?id=36e093a31a9eb12021e7780b9e322c29763ffa58
These patches to configure.ac and gio/Makefile.am are not used by the port but are
used by the maintainer to regenerate the patches to configure and gio/Makefile.in
that are used by the port.
--- configure.ac.orig	2015-09-21 07:38:33.000000000 -0500
+++ configure.ac	2015-10-10 18:32:37.000000000 -0500
@@ -223,15 +223,6 @@
   AC_DEFINE(HAVE_COCOA, 1, [define to 1 if Cocoa is available])
   COCOA_LIBS="-Wl,-framework,Foundation"
   LDFLAGS="$LDFLAGS $COCOA_LIBS"
-
-  osx_version=`sw_vers -productVersion`
-  osx_min_version="10.9.0"
-  AC_MSG_CHECKING([OSX version >= $osx_min_version])
-  AS_VERSION_COMPARE([$osx_version], [$osx_min_version], [
-    AC_MSG_RESULT([no])
-    AC_MSG_ERROR([OSX version is too old!])
-  ])
-  AC_MSG_RESULT([yes])
 ], [COCOA_LIBS=""])
 
 AC_SUBST([COCOA_LIBS])
--- gio/Makefile.am.orig	2015-09-04 12:56:56.000000000 -0500
+++ gio/Makefile.am	2015-10-10 18:32:38.000000000 -0500
@@ -265,9 +265,6 @@
 	ggtknotificationbackend.c \
 	$(NULL)
 
-if OS_COCOA
-unix_sources += gcocoanotificationbackend.c
-endif
 
 giounixincludedir=$(includedir)/gio-unix-2.0/gio
 giounixinclude_HEADERS = \
@@ -527,7 +524,7 @@
 if OS_COCOA
 # This is dumb.  The ObjC source file should be properly named .m
 libgio_2_0_la_CFLAGS += -xobjective-c
-libgio_2_0_la_LDFLAGS += -Wl,-framework,Foundation -Wl,-framework,AppKit
+libgio_2_0_la_LDFLAGS += -Wl,-framework,Foundation
 endif
 
 libgio_2_0_la_DEPENDENCIES = $(gio_win32_res) $(gio_def) $(platform_deps)
